examples/locality-load-balancing/verify.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash -e
+
+export NAME=locality-load-balancing
+export DELAY=5
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+check_health() {
+    docker-compose exec -T client-envoy curl -s localhost:8001/clusters | grep health_flags
+}
+
+request_backend() {
+    docker-compose exec -T client-envoy python3 client.py http://localhost:3000/ 100
+}
+
+bring_up_backend() {
+    local server
+    server="$1"
+
+    docker-compose exec -T client-envoy curl -s "$server":8000/healthy
+}
+
+bring_down_backend() {
+    local server
+    server="$1"
+
+    docker-compose exec -T client-envoy curl -s "$server":8000/unhealthy
+}
+
+run_log "=== Demo setup
+client  -> backend-local-1      [priority: 0, weight: 1]
+        -> backend-local-2      [priority: 1, weight: 1]
+        -> backend-remote-1     [priority: 1, weight: 1]
+        -> backend-remote-2     [priority: 2, weight: 1]
+"
+
+run_log "=== Scenario 1: one replica in the highest priority locality"
+
+run_log "Send requests to backend."
+check_health
+request_backend
+
+run_log "Bring down backend-local-1 then snooze for ${DELAY}s. Priority 0 locality is 0% healthy."
+bring_down_backend "${NAME}"_backend-local-1_1
+sleep ${DELAY}
+
+run_log "Send requests to backend."
+check_health
+request_backend
```

Comment:
it would be good to grep the responses here to check that all of the expected backends respond

Reply:
the client app does print out the responding endpoints, like this:

```
Hello from backend-local-2!: actual weight 34.0%
Hello from backend-remote-1!: actual weight 34.0%
Hello from backend-local-1!: actual weight 32.0%
```

Reply:
yep, i was wondering how we can grep for multiple things - it might be easiest to capture the output and grepping in that

either way, i think its important to test the outputs

Reply:
Added a `check_backend()` in verify script, so that it will stop the process if expected backend is missing from the response.